File: app-core/src/main/java/code/sdk/core/manager/AdjustLifecycleCallbacks.java
package code.sdk.core.manager;

import android.app.Activity;
import android.app.Application;
import android.os.Bundle;

import com.adjust.sdk.Adjust;

import code.util.LogUtil;

public class AdjustLifecycleCallbacks extends SimpleLifecycleCallbacks {

    private static final String TAG = AdjustLifecycleCallbacks.class.getSimpleName();

    @Override
    public void onActivityResumed(Activity activity) {
        Adjust.onResume();
    }

    @Override
    public void onActivityPaused(Activity activity) {
        Adjust.onPause();
    }

}

File: app-core/src/main/java/code/sdk/core/manager/AdjustLifecycleCallbacks.kt
package code.sdk.core.manager

import android.app.Activity
import com.adjust.sdk.Adjust

class AdjustLifecycleCallbacks : SimpleLifecycleCallbacks() {

    private val TAG = AdjustLifecycleCallbacks::class.java.simpleName
    override fun onActivityResumed(activity: Activity) {
        Adjust.onResume()
    }

    override fun onActivityPaused(activity: Activity) {
        Adjust.onPause()
    }
}
